src/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -361,432 +363,383 @@ const SendAssetsForm = ({ txConfig, onComplete }: IStepComponentProps) => {
+                  getBaseAssetByNetwork({ network, assets: userAssets }) || ({} as Asset)
+                );
               }
             }
-          }, [values.address.value]);
+          }}
+        />
+      </fieldset>
+      {/* Sender Address */}
+      <fieldset className="SendAssetsForm-fieldset">
+        <label htmlFor="account" className="input-group-header">
+          <div>
+            {translate('X_SENDER')} <Tooltip tooltip={translateRaw('SENDER_TOOLTIP')} />
+          </div>
+        </label>
 
-          const toggleAdvancedOptions = () => {
-            setFieldValue('advancedTransaction', !values.advancedTransaction);
-          };
-          const toggleIsAutoGasSet = () => {
-            // save value because setFieldValue method is async and values are not yet updated	            // save value because setFieldValue method is async and values are not yet updated
-            const isEnablingAutoGas = !values.isAutoGasSet;
-            setFieldValue('isAutoGasSet', !values.isAutoGasSet);
-            if (isEnablingAutoGas) {
-              handleGasEstimate(true);
-            }
-          };
-          const handleGasEstimate = async (forceEstimate: boolean = false) => {
-            if (
-              values &&
-              values.network &&
-              values.asset &&
-              values.address &&
-              isValidETHAddress(values.address.value) &&
-              values.account &&
-              isValidPositiveNumber(values.amount) &&
-              (values.isAutoGasSet || forceEstimate)
-            ) {
-              setIsEstimatingGasLimit(true);
-              const finalTx = processFormForEstimateGas(values);
-              const gas = await getGasEstimate(values.network, finalTx);
-              setFieldValue('gasLimitField', hexToNumber(gas).toString());
+        <AccountSelector
+          name="account"
+          value={values.account}
+          accounts={accountsWithAsset}
+          onSelect={(account: StoreAccount) => {
+            setFieldValue('account', account); //if this gets deleted, it no longer shows as selected on interface, would like to set only object keys that are needed instead of full object
+            handleGasEstimate();
+          }}
+          asset={values.asset}
+        />
+      </fieldset>
+      <fieldset className="SendAssetsForm-fieldset">
+        <label htmlFor="address" className="input-group-header">
+          {translate('X_RECIPIENT')}
+        </label>
+        <ContactLookupField
+          name="address"
+          value={values.address}
+          error={errors && touched.address && errors.address && (errors.address as ErrorObject)}
+          network={values.network}
+          isResolvingName={isResolvingName}
+          onBlur={handleGasEstimate}
+          setIsResolvingDomain={setIsResolvingDomain}
+          setFieldValue={setFieldValue}
+          setFieldTouched={setFieldTouched}
+          setFieldError={setFieldError}
```

Comment:
I suspect that this can be reduced to `handleChange` https://formik.org/docs/api/formik#handlechange-e-reactchangeeventany--voidv

Reply:
Due to complexity of `GeneralLookupField` let's handle later.